harvard-lts/etd_alma_service
# ETD-272: Add existing scripts -Generate MARCXML

**Add existing scripts -Generate MARCXML**
* * *

**JIRA Ticket**: https://jira.huit.harvard.edu/browse/ETD-272


# What does this Pull Request do?
this copies etd mets into marcxml for alma

# How should this be tested?

0) copy .env.example to .env, set it to dev values
1) spin up instance locally
2) enter container
3) `pytest` - all tests should pass


# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? y
- integration tests? n

# Interested parties
Tag (@ mention) interested parties: @awoods
